### PR TITLE
core: services: kraken: Add size of extension

### PR DESCRIPTION
**Pull request by patrickelectric**
image
ps: Yes, it's a 128GB sdcard

### CODE REVIEW

**Comment by patrickelectric** [comment]
ping @joaoantoniocardoso @Williangalvani

File: core/services/kraken/kraken.py [hunk 1]
```diff
@@ -310,11 +315,18 @@ async def load_stats(self) -> Dict[str, Any]:
             except KeyError:
                 memory_usage = "N/A"
 
+            try:
+                print(show)
```
Comment (Williangalvani) [inline core/services/kraken/kraken.py:319]:
```suggestion
```
debugging code

**Review by Williangalvani — CHANGES_REQUESTED** [review]
it misbehaved a couple of times here, where it only loaded stats for some of the containers, but I don't think it is related to this pr.

Other than that print, it looks great

**Review by joaoantoniocardoso — APPROVED** [review]

File: core/frontend/src/components/kraken/InstalledExtensionCard.vue [hunk 2]
```diff
@@ -199,6 +223,14 @@ export default Vue.extend({
...
       return total_kb ?? undefined
     },
+    main_disk(): number | undefined {
+      const disks = system_information.system?.disk
+      return disks?.find((sensor) => sensor.mount_point === '/')
+    },
+    main_disk_size(): number {
+      const value = this.main_disk?.total_space_B ?? 0
+      return value / 1024 // Move to kB
+    },
```
Comment (Williangalvani) [inline core/frontend/src/components/kraken/InstalledExtensionCard.vue:233]: My brain is not working well today. but the typing here looks wrong.

if `main_disk()` is a number, how can we access `total_space_B` of it in the second method?

And why is CI not catching this?

**Review by Williangalvani — CHANGES_REQUESTED** [review]
sorry, minor typing issue

**Comment by patrickelectric** [comment]
ping @Williangalvani

**Pull request merged**